### PR TITLE
SPU2: Remove user options for disabling reverb and dealias filter

### PR DESCRIPTION
**Pull request by MrCK1**
### Description of Changes
Remove options that are no longer necessary or useful for users.

### Rationale behind Changes
Code quality/simplifcation, less options for enduser to worry about

### Suggested Testing Steps
Open SPU2, test sound output to ensure functionality.

### CODE REVIEW

File: pcsx2/SPU2/Windows/SPU2.rc [hunk 1]
```diff
@@ -186,6 +181,17 @@ BEGIN
...
+/////////////////////////////////////////////////////////////////////////////
+//
+// AFX_DIALOG_LAYOUT
+//
+
+IDD_CONFIG AFX_DIALOG_LAYOUT
+BEGIN
+    0
+END
+
```
Comment (refractionpcsx2) [inline pcsx2/SPU2/Windows/SPU2.rc:194]: Where'd this come from?

Comment (RedDevilus) [reply pcsx2/SPU2/Windows/SPU2.rc:194]: If you use the vs designer it will auto-create lines like that.

Comment (refractionpcsx2) [reply pcsx2/SPU2/Windows/SPU2.rc:194]: I figured that, thanks Red. It needs to be removed

Comment (MrCK1) [reply pcsx2/SPU2/Windows/SPU2.rc]: Thanks, I took care of it.

**Comment by refractionpcsx2** [comment]
LGTM, but can you align everything in the left box? It's triggering me that the interpolation label and dropdown are more left.  Can you shrink it enough to push it over without cutting the text off?

**Pull request merged**